src/api/module/VBClubMgmt/src/V1/Rest/Api/EventEntityInterface.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+namespace VBClubMgmt\V1\Rest\Api;
+
+interface EventEntityInterface
+{
+    const ID = 'event_id';
+    const NAME = 'name';
+    const DATE = 'date';
+    const DESCRIPTION = 'description';
+    const PARTICIPANTS = 'participants';
+    const DATA = 'data';
+    const CREATED_AT = 'created_at';
+    const UPDATED_AT = 'updated_at';
+
+    /***
+     * @return int
+     */
+    public function getEventId(): int;
+
+    /**
+     * @param int $eventId
+     *
+     * @return \VBClubMgmt\V1\Rest\Api\EventEntityInterface
+     */
+    public function setEventId($eventId): EventEntityInterface;
+
+    /***
+     * @return string
+     */
+    public function getName(): ?string;
+
+    /**
+     * @param string $name
+     *
+     * @return \VBClubMgmt\V1\Rest\Api\EventEntityInterface
+     */
+    public function setName($name): EventEntityInterface;
+
+    /***
+     * @return string
+     */
+    public function getDate(): ?string;
+
+    /**
+     * @param string $date
+     *
+     * @return \VBClubMgmt\V1\Rest\Api\EventEntityInterface
+     */
+    public function setDate($date): EventEntityInterface;
+
+    /***
+     * @return string
```

Comment:
According to the return type below, this should be 

```suggestion
     * @return string|null
```

Reply:
I'm not really convinced of the utility to specified null type return on getters.
But i'm keeping this point in mind